docs/csharp/language-reference/operators/null-parameter-check.md

```diff
@@ -69,7 +69,7 @@ When you call that method passing in a null argument as shows in the following e
 
 :::code language="csharp" source="./snippets/shared/NullParameterCheck.cs" id="Enumerate":::
 
-The message "Call iterator" is displayed before the exception is thrown. That's because the null check doesn't run until the first element in the sequence is requested. Instead, if you use the `!!` operator, the exception is thrown when `CharsIn` is called, not when the first element of the sequence is requested.
+The message "Enumerate values" is displayed before the exception is thrown. That's because the null check doesn't run until the first element in the sequence is requested. Instead, if you use the `!!` operator, the exception is thrown when `CharsIn` is called, not when the first element of the sequence is requested.
```

Comment:
```suggestion
The message ":::no-loc text="Enumerate values":::" is displayed before the exception is thrown. That's because the null check doesn't run until the first element in the sequence is requested. Instead, if you use the `!!` operator, the exception is thrown when `CharsIn` is called, not when the first element of the sequence is requested.
```

This prevents the text from being translated. From current French site:

![image](https://user-images.githubusercontent.com/31348972/163120825-ac083d45-3938-4c3a-85e3-d2a2bc276bc1.png)